### PR TITLE
Added new SwiftUI wrappers.

### PR DESCRIPTION
**Pull request by blackrain**

### CODE REVIEW

File: Shared/SwiftUI/SwiftUIView.swift [hunk 1]
```diff
@@ -4,42 +4,76 @@ import SwiftUI
...
                     if case let .contentDidChange(cGSize) = action {
-                        gridHeight = cGSize.height
+                        storiesGridHeight = cGSize.height
                     }
+                })
+                .frame(height: storiesGridHeight)
+                
+                Text("SwiftUI Clips RowView")
+                    .padding(.leading, 8)
+                StorytellerClipsRow(configuration: model.clipsRowConfiguration) { _ in
```
Comment (stodirascu) [inline Shared/SwiftUI/SwiftUIView.swift:53]: Would be good to make this block optional, so we don't have to write if if we don't need it.
cc @blackrain

File: Shared/MainFlowController.swift [hunk 2]
```diff
@@ -68,8 +68,6 @@ final class MainFlowController {
 
     func navigateToSwiftUI(cellType: StorytellerListViewCellType, delegate: StorytellerListDelegate?) {
```
Comment (bartoszstelmaszuk) [inline Shared/MainFlowController.swift:69]: No need to pass those parameters if they are not used inside function :)

**Review by bartoszstelmaszuk — APPROVED** [review]
Good work! One comment

File: Shared/MainFlowController.swift (hunk 2 above)
Comment (blackrain) [reply Shared/MainFlowController.swift]: Thanks @bartoszstelmaszuk, updated.

File: Shared/SwiftUI/SwiftUIView.swift (hunk 1 above)
Comment (blackrain) [reply Shared/SwiftUI/SwiftUIView.swift]: @stodirascu made it optional.

**Review by stodirascu — APPROVED** [review]
Perfect

**Review by fnabrdal — APPROVED** [review]
Good work!

**Pull request merged**